src/Style/TerminusStyle.php

```diff
@@ -23,7 +21,6 @@ class TerminusStyle extends SymfonyStyle
      */
     public function __construct(InputInterface $input, OutputInterface $output)
```

Comment:
It doesn't seem this function anymore. Is it safe to remove?

Reply:
Yes, it could be removed. That would leave this class completely empty, and it could be completely removed as well. Perhaps some of the content here might be desired as a placeholder, but I'll take out as much as you want.